### PR TITLE
Drop Doctrine\DBAL\Version in favor of Ocramius\PackageVersions

### PR DESCRIPTION
**Pull request by Majkl578**
cross-linking doctrine/doctrine2#6653

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": "^7.1",
         "ext-pdo": "*",
-        "doctrine/common": "^2.7.1"
+        "doctrine/common": "^2.7.1",
+        "ocramius/package-versions": "^1.2"
```
Comment (morozov) [inline composer.json]: I know it's already merged into `doctrine/doctrine2` but given the version is displayed in CLI to humans, would it make sense to use `jean85/pretty-package-versions` for prettier output?

Comment (Ocramius) [reply composer.json]: Eh, this is mostly so we can get proper reports when people get something wrong, so I think the human consumer is mostly @doctrine/doctrinecore

Comment (Majkl578) [reply composer.json]: Agreed, I would keep it simple and change if requested by users eventually, even now it's 2x more dependencies than before. 😃

**Comment by Ocramius** [comment]
Manually merged via doctrine/dbal@2f085fc00d1a70e2e8fb7298c496bc9a58c68f79

**Pull request closed without merging**